r/tools/nixlibs.R

```diff
@@ -396,6 +401,7 @@ cmake_version <- function(cmd = "cmake") {
 with_s3_support <- function(env_vars) {
   arrow_s3 <- toupper(Sys.getenv("ARROW_S3")) == "ON" || tolower(Sys.getenv("LIBARROW_MINIMAL")) == "false"
   if (arrow_s3) {
+    # TODO: add in brew versions of these?
```

Comment:
Please do

Reply:
I'm working on this, it ends up being a bit more complicated than simply `brew install`ing, but I'll get something working.

Reply:
If it's trouble then don't bother, this is a rather esoteric case isn't it? Also it seems pretty likely that you would already have openssl and curl (I do and I didn't install them myself).